marimbabot_description/launch/marimbabot_ur5_upload.launch

```diff
@@ -1,5 +1,17 @@
 <?xml version="1.0"?>
 <launch>
-  <param name="robot_description" command="xacro '$(find marimbabot_description)/urdf/marimbabot_ur5.urdf.xacro'
-    joint_ranges_config:=$(find tams_ur5_description)/config/joint_ranges/elbow_up.yaml" />
+  <arg name="joint_ranges_config" default="$(find tams_ur5_description)/config/joint_ranges/elbow_up.yaml" />
+  <arg name="floating_table" default="false" />
+  <arg name="r200" default="false" />
```

Comment:
What param is r200?

Reply:
It was the mounted camera, removed for now. thanks